git/commits.go

```diff
@@ -29,7 +29,7 @@ func GetConventionalCommitTypesSinceLastRelease(repository *git.Repository) (Con
 		}
 		return ConventionalCommmitTypesResult{}, err
 	}
-	commitIterator, err := repository.Log(&git.LogOptions{From: head.Hash()})
+	commitIterator, err := repository.Log(&git.LogOptions{From: head.Hash(), All: true, Order: git.LogOrderCommitterTime})
```

Comment:
According to the [documentation of the `go-git` module](https://pkg.go.dev/github.com/go-git/go-git/v5#LogOptions) using `From` and `Order` at the same time is incompatible. I.e., if `Order` is being used, `From` is ignored. So we should remove one of these. If it's actually needed, we should remove `From`. If it is not needed, we should remove it.

Can you please double-check this?